Performance/Harness.swift

```diff
@@ -14,6 +14,14 @@
 
 import Foundation
 
+private func padded(_ input: String, to width: Int) -> String {
+    var s = input
```

Comment:
Instead of a loop, what about reducing this to something like...
```
return input + String(repeating: " ", count: max(0, width - input.characters.count))
```

(Not actually verified, maybe typos.)